### ISSUE TITLE
	 [ANN] [1080 | 1080TI] ETHlargement - The Hashrate Hardener

### ISSUE THREAD

**Issue by jjziets** [nicehash/NiceHashMiner#1201]
Is there away to get ETHlargement to work?

https://bitcointalk.org/index.php?topic=3370685.0;all

The problem is that the mod needs to be done before the miners start with no OC and after the miner runs the OC can be applied.

**Comment by S74nk0** [comment]
Issue related to old nhm version/build. Please use latest version and open a new issue for bugs and/or features.
